tendrl-monitoring-integration.spec

```diff
@@ -89,6 +88,9 @@ if [ "$1" = 0 ] ; then
 fi
 %systemd_preun tendrl-monitoring-integration.service
 
+%postun
+firewall-cmd --reload >/dev/null 2>&1 || :
```

Comment:
why reload is done at two places?(line 80)

Reply:
for before and after install

Reply:
ok